contracts/extensions/VotingReputation/VotingReputationMisalignedRecovery.sol

```diff
@@ -0,0 +1,225 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.7.3;
+pragma experimental ABIEncoderV2;
+
+import "./../../colonyNetwork/IColonyNetwork.sol";
+import "./../../colony/ColonyRoles.sol";
+import "./../../common/BasicMetaTransaction.sol";
+import "./../../common/ERC20Extended.sol";
+import "./../../patriciaTree/PatriciaTreeProofs.sol";
+import "./../../tokenLocking/ITokenLocking.sol";
+import "./../ColonyExtension.sol";
+import "./../../../lib/dappsys/math.sol";
+import "./VotingReputationDataTypes.sol";
+
+contract VotingReputationMisalignedRecovery is PatriciaTreeProofs, DSMath, DSAuth, VotingReputationDataTypes {
+
+  // THIS FILE IS DELIBERATELY WRONG. IF YOU'RE EDITING THIS FILE, AND YOU'VE NOT BEEN EXPLICITLY
+  // TOLD TO DO SO, LEAVE NOW. THERE BE DRAGONS HERE.
+
+  // Constants
+  uint256 constant UINT128_MAX = 2**128 - 1;
+
+  uint256 constant NAY = 0;
+  uint256 constant YAY = 1;
+
+  address resolver; // Align storage with EtherRouter
+
+  IColony colony;
+  bool deprecated;
+
+  ExtensionState state;
+
+  IColonyNetwork colonyNetwork;
+  ITokenLocking tokenLocking;
+  address token;
+
+  // All `Fraction` variables are stored as WADs i.e. fixed-point numbers with 18 digits after the radix. So
```

Comment:
I would delete all the variable descriptive comments in this file, it gives the impression that this file is to be "taken seriously"